### ISSUE TITLE
about the loss

### ISSUE THREAD

**Issue by Elegantice** [Sudo-Biao/s-LWSR#3]
sorry, maybe i dont read your article in detail. i cant find the introduce about the loss.And your code has much loss. i think they are the different loss of different model. can u give me a introduce for your loss? thank u very much.

**Comment by Elegantice** [comment]
In detail,i can not find the out, out2, out 3, out 4.just can find the sr.so idont have the complete loss.because i cant get the loss1, loss2,loss3. The reason is that I can only get one result from the self.model(lr, idx_scale).look forward to your reply.

**Comment by Feynman1999** [comment]
@Elegantice I also don't know. I guss L1 loss? but i can't reproduce the results in paper...

**Comment by Sudo-Biao** [comment]
L 1 loss, and you can check with the pre-trained model , or send me your problem.发自我的华为手机-------- 原始邮件 --------发件人： Yuxiang Chen <notifications@github.com>日期： 2020年1月6日周一 傍晚5:27收件人： Sudo-Biao/s-LWSR <s-LWSR@noreply.github.com>抄送： Subscribed <subscribed@noreply.github.com>主    题： Re: [Sudo-Biao/s-LWSR] about the loss (#3)@Elegantice I also don't know. I guss L1 loss? but i can't reproduce the results in paper...

**Comment by Feynman1999** [comment]
thanks for your reply! I have some questions below:
*I found in your code that the input images is not divided by 255. Is there
any difference between this and dividing by 255 to map to 0 ~ 1?
I think there is no difference, so my code is mapped to 0 ~ 1.

* What has data enhancement done specifically? How much do these actions
affect the final result?

* The value range of the output image is not exactly 0-255, is it 0 if it
is less than 0, or 255 If it is greater than 255 (clamp operation)?

* How many epochs are trained by the results shown in the paper?

Thank you for taking the time to read and give advice.

yuxiang chen

Biao <notifications@github.com> 于2020年1月6日周一 下午5:44写道：

**Comment by Sudo-Biao** [comment]
> sorry, maybe i dont read your article in detail. i cant find the introduce about the loss.And your code has much loss. i think they are the different loss of different model. can u give me a introduce for your loss? thank u very much.

Sorry for the mistake. I update the right code now.

**Comment by Sudo-Biao** [comment]
> @Elegantice I also don't know. I guss L1 loss? but i can't reproduce the results in paper...

Sorry for the mistake, and I have updated the right code. You can try it.
